### DISCUSSION TITLE
Neptune 2 Marlin 2.1.2 elegoo_pic files

### DISCUSSION THREAD

**Discussion by TheFeralEngineer** [NARUTOfzr/ZNP-Robin-Nano-V1.2#10] [Q&A]
Is there any particular reason why the elegoo_pic files will not work on Marlin 2.1.2 with lvgl?

**Comment by NARUTOfzr** [comment]
I tried this a long time ago and most of the image information will not work on marlin2. I guess that would require re-converting the image information:https://lvgl.io/tools/imageconverter
